python/whylogs/core/resolvers.py

```diff
@@ -122,47 +122,79 @@ def __post_init__(self):
             raise ValueError("ResolverSpec: resolver specification column type must be a DataType")
 
 
-# whylabs expects COLUMN_METRICS to be present for every column.
-COLUMN_METRICS = [MetricSpec(StandardMetric.counts.value), MetricSpec(StandardMetric.types.value)]
+@dataclass
+class AntiResolver(ResolverSpec):
```

Comment:
Maybe "Filter" or "Exclusion" resolver is closer to describing what this does, because its still resolving a column type to a profiling behavior. Not sure what a good name is.